pkg/client/request.go

```diff
@@ -110,6 +110,8 @@ type Request struct {
 	selector     labels.Selector
 	timeout      time.Duration
 
+	apiVersion string
```

Comment:
This does not seem to be used anywhere.
Is this change required? Or am I missing something.